tests/security/selinux/selinux_setup.pm

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Setup environment for selinux test
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#40358
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+use version_utils "is_opensuse";
+
+sub run {
+    select_console "root-console";
+
+    if (is_opensuse) {
+        zypper_call("ar http://download.opensuse.org/tumbleweed/repo/oss/ tumbleweed-Oss");
+    }
+
+    # program 'sestatus' can be found in policycoreutils pkgs
+    zypper_call("--no-refresh in policycoreutils");
+
+    my $pkgs = script_output("echo `zypper --no-refresh se selinux | grep -i selinux | grep -v -w srcpackage | cut -d '|' -f 2`");
```

Comment:
"just give me everything what has selinux in name" - does not look like good idea. It can go wrong in many different ways . Vote for having closed list of packages which we want to cover, or maybe there is some selinux pattern which we can install ?

Reply:
1. deleted "--no-refresh"
2. untouched line "36":
from my understanding, selinux is not perfectly supported in suse products now,
and there is no selinux pattern to install, also selinux related packages vary in different products.
So, I prefer "zypper se ..." instead of a closed list of selinux packages.